ProteoformSuiteInternal/Protein.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Proteomics;
+
+namespace ProteoformSuiteInternal
+{
+    public class ProteinSequenceGroup : Protein
```

Comment:
I think I renamed this class ProteinSequenceGroup. It should inherit from ProteinWithGoTerms.